pkg/testcase/secretsencrypt.go

```diff
@@ -0,0 +1,158 @@
+package testcase
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestSecretsEncryption() {
+	nodes, errGetNodes := shared.GetNodesByRoles("etcd", "control-plane")
+	Expect(nodes).NotTo(BeEmpty())
+	Expect(errGetNodes).NotTo(HaveOccurred(), "error getting etcd/control-plane nodes")
+
+	product, err := shared.Product()
+	Expect(err).NotTo(HaveOccurred(), "error getting product from config")
+
+	errSecret := shared.CreateSecret("secret1", "default", false)
+	Expect(errSecret).NotTo(HaveOccurred(), "error creating secret")
+
+	shared.LogLevel("INFO", "TEST: 'CLASSIC' Secrets Encryption method")
```

Comment:
`INFO` Should be `info` as defined in the LogLevel func

Reply:
fixed